RELEASE-NOTES.txt

```diff
@@ -3,6 +3,7 @@
 - Enhancement: The Reviews tab now presents all the Product Reviews
 - bugfix: now Order Detail should not have some dark cells when launching the app in Dark mode.
 - bugfix: fixed UI appearance on cells of Order List when tappen with dark mode enabled.
+- bugfix: fixed UI appearance of reviews with dark mode enabled.
```

Comment:
What do you think about modifying the description? I try to avoid "fixed" because it doesn't tell the user how we fixed it.

```
bugfix: Reviews no longer convert to partial dark mode. Dark mode coming soon!
```